src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetUnallocatedCasesByTeamTests.kt

```diff
@@ -378,6 +406,11 @@ class GetUnallocatedCasesByTeamTests : IntegrationTestBase() {
       ),
     )
 
+    workforceAllocationsToDelius.setExcludedUsersByCrn(
+      listOf("J678910", "J680660", "X4565764", "J680648", "X6666222", "XXXXXXX", "ZZZZZZZ", "AB77711"),
+    )
+    workforceAllocationsToDelius.setApopUsers()
+
     webTestClient.get()
       .uri("/team/TEAM1/cases/unallocated")
       .headers { it.authToken(roles = listOf("ROLE_MANAGE_A_WORKFORCE_ALLOCATE")) }
```

Comment:
Suggest to add a test to specify new behaviour of getting excluded cases

Reply:
done, test shows an excluded, an excluded that has an Apop jser  and ignores the remaining  restricted cases